utils/src/ooinstall/ansible_plugins/facts_callback.py

```diff
@@ -27,9 +27,19 @@ def on_any(self, *args, **kwargs):
     def runner_on_failed(self, host, res, ignore_errors=False):
         pass
 
-    def runner_on_ok(self, host, res):
-        if res['invocation']['module_args'] == 'var=result':
-            facts = res['var']['result']['ansible_facts']['openshift']
+    def runner_on_ok(self, *args):
```

Comment:
This is a bad hack - there is probably a more elegant way to have the callback class work with both ansible 1.x/2.0/2.1